libgloss/arc/hl/hl_gw.c

```diff
@@ -0,0 +1,158 @@
+/*********************************************************************
+Copyright  2002-2020 Synopsys Inc.
+*********************************************************************/
+
+#include "hl_gw.h"
+
+#define HL_VERSION 1
+
+
+/* Maximum message size without service information, sa HL_BUF_RESERVED. */
+#ifdef HL_IOCHUNK
+  #pragma message("Using HL_IOCHUNK as hostlink buffer size")
```

Comment:
don;t know what is this. No pragma please

Reply:
done